### PR TITLE
Check format on CI

### PR DESCRIPTION
**Pull request by TucksonDev**
Closes TW-277

### CODE REVIEW

**Comment by TucksonDev** [comment]
Fails if files are not properly formatted: https://github.com/OffchainLabs/arbitrum-docs/actions/runs/11667154963/job/32484000386?pr=1789

**Comment by TucksonDev** [comment]
Succeeds if files are formatted: https://github.com/OffchainLabs/arbitrum-docs/actions/runs/11667283684/job/32484241069?pr=1789

File: arbitrum-docs/bold/concepts/bold-technical-deep-dive.md [hunk 1]
```diff
@@ -203,8 +202,11 @@ To create a challenge, there must be a fork in the Arbitrum assertion chain smar
 
 <a data-quicklook-from="challenge">Challenge</a> bonds are named as such because they are bonds required
 for opening challenges. The mechanism of how challenge bond economics are decided is contained in the
-[Economics of Disputes](../../how-arbitrum-works/bold/bold-economics-of-disputes.md), which also explains the cost profile and spam prevention in BoLD. In short, the
-actual cost of a bond encompasses many costs associated with participating in the dispute game. More information on the bond sizes and how they were calculated can be found in the [Economics of Disputes](../../how-arbitrum-works/bold/bold-economics-of-disputes.md) document mentioned above.
+[Economics of Disputes](../../how-arbitrum-works/bold/bold-economics-of-disputes.md), which also explains
+the cost profile and spam prevention in BoLD. In short, the actual cost of a bond encompasses many costs
+associated with participating in the dispute game. More information on the bond sizes and how they were
+calculated can be found in the [Economics of Disputes](../../how-arbitrum-works/bold/bold-economics-of-disputes.md)
+document mentioned above.
```
Comment (anegg0) [inline arbitrum-docs/bold/concepts/bold-technical-deep-dive.md:209]: Did the formater add a carriage return, here?

**Review by anegg0 — APPROVED** [review]
Looking like $1000000 to me!

**Pull request merged**